JASP-Engine/JASP/R/multinomialtestbayesian.R

```diff
@@ -0,0 +1,446 @@
+    multinomialResults$mainTable[[nms[h]]] <- .multBayesBfEquality(alphas = a, counts = t, thetas = hyps[[h]])
+  }
+  multinomialResults$mainTable[["prior"]]   <- a
+  multinomialResults$mainTable[["levels"]]  <- levels(fact)
+  multinomialResults$mainTable[["nlevels"]] <- length(levels(fact))
+  multinomialResults$mainTable[["hypNames"]]<- nms
+  multinomialResults$mainTable[["nhyps"]]   <- nhyps
+
+  #  Results for descriptives plot
+  multinomialResults$descriptivesPlot[["descProps"]]  <- .multComputeCIs(t, options$descriptivesPlotCredibleInterval)
+  multinomialResults$descriptivesPlot[["descCounts"]] <- multinomialResults$descriptivesPlot[["descProps"]] * N
+  multinomialResults$descriptivesPlot[["descProps"]][["observed"]]   <- as.numeric(t)/N
+  multinomialResults$descriptivesPlot[["descCounts"]][["observed"]] <- as.numeric(t)
+  multinomialResults$descriptivesPlot[["descProps"]][["fact"]]  <- levels(fact)
+  multinomialResults$descriptivesPlot[["descCounts"]][["fact"]] <- levels(fact)
+
+  # Results for descriptives table
+  multinomialResults$descriptivesTable[["descProps"]][["fact"]]     <- levels(fact)
+  multinomialResults$descriptivesTable[["descCounts"]][["fact"]]    <- levels(fact)
+  multinomialResults$descriptivesTable[["descProps"]][["observed"]]  <- as.numeric(t)/N
+  multinomialResults$descriptivesTable[["descCounts"]][["observed"]] <- as.numeric(t)
+  for(h in 1:nhyps){
+    multinomialResults$descriptivesTable[["descProps"]][[nms[h]]]    <- multinomialResults$mainTable[[nms[h]]]$expected/N
+  }
+    for(h in 1:nhyps){
+    multinomialResults$descriptivesTable[["descCounts"]][[nms[h]]]   <- multinomialResults$mainTable[[nms[h]]]$expected
+    }
+
+  multinomialResults$descriptivesTable[["descProps"]]    <- setNames(as.data.frame(multinomialResults$descriptivesTable[["descProps"]]), c("fact", "observed", nms))
+  multinomialResults$descriptivesTable[["descCounts"]]   <- setNames(as.data.frame(multinomialResults$descriptivesTable[["descCounts"]]), c("fact", "observed", nms))
+  multinomialResults$descriptivesTable[["descPropsCI"]]  <- .multComputeCIs(t, options$credibleIntervalInterval)
+  multinomialResults$descriptivesTable[["descCountsCI"]] <- .multComputeCIs(t, options$credibleIntervalInterval) * N
+
+  # Save results to state
+  defaultOptions <- multinomialResults$specs$defaultOptions
+  jaspResults[["stateMultinomialBayesianResults"]] <- createJaspState(multinomialResults)
+  jaspResults[["stateMultinomialBayesianResults"]]$dependOnOptions(defaultOptions)
+
+  return(multinomialResults)  # return the out object
+}
+.createMultBayesMainTable <- function(jaspResults, options, multinomialResults){
+  if (!is.null(jaspResults[["multinomialTable"]])) return()
+
+  # Create table
+  defaultOptions                    <- multinomialResults$specs$defaultOptions
+  multinomialTable                  <- createJaspTable(title = "Bayesian Multinomial Test")
+  jaspResults[["multinomialTable"]] <- multinomialTable
```

Comment:
Add `multinomialTable` to `jaspResults` after you add all the `$addColumnInfo`, otherwise the output doesn't show the empty table properly